react-form-controls/src/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react"
+import ReactDOM from "react-dom"
+
+class NewsletterForm extends React.Component{
+  constructor(props){
+    super(props)
+    this.handleChange=this.handleChange.bind(this)
+    this.handleSubmit=this.handleSubmit.bind(this)
+    this.state={email:""}
+  }
+
+  handleChange(event){
+    this.setState({email:event.target.value})
+  }
+
+  handleSubmit(event){
+    event.preventDefault()
+    console.log("state:",this.state)
+  }
+
+  render(){
+    return (
+      <form onSubmit={this.handleSubmit}>
+        <label for="email">Email</label>
+        <input onChange={this.handleChange} type="text" id="email" required/>
```

Comment:
you need to add the value attribute here and assign it the value stored in state so that React can maintain control of the inputs display on the DOM